api/tests/core/offers/test_api.py

```diff
@@ -2095,7 +2102,7 @@ def test_deactivate(self, caplog):
 class ActivateFutureOffersTest:
     @mock.patch("pcapi.core.search.async_index_offer_ids")
     def test_activate_future_offers_empty(self, mocked_async_index_offer_ids):
-        offer = factories.OfferFactory(isActive=False)  # Offer not in the future, i.e. no publication_date
+        offer = factories.OfferFactory(publicationDatetime=None)  # Offer not in the future, i.e. no publication_date
```

Comment:
Détail (pas ton code) : le commentaire est un peu mensonger
```suggestion
        offer = factories.OfferFactory(publicationDatetime=None)  # Offer not in the future, e.g. draft or paused (no publication_date)